<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GenerateReportsWidget</class>
 <widget class="QWidget" name="GenerateReportsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>455</width>
    <height>145</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>DejaVu Sans</family>
    <pointsize>8</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>5</number>
   </property>
   <property name="leftMargin">
    <number>20</number>
   </property>
   <property name="topMargin">
    <number>7</number>
   </property>
   <property name="rightMargin">
    <number>20</number>
   </property>
   <property name="bottomMargin">
    <number>7</number>
   </property>
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>2</number>
      </property>
      <property name="leftMargin">
       <number>9</number>
      </property>
      <property name="topMargin">
       <number>5</number>
      </property>
      <property name="rightMargin">
       <number>5</number>
      </property>
      <property name="bottomMargin">
       <number>5</number>
      </property>
      <item>
       <widget class="QRadioButton" name="rdGenerateReportsForContaining">
        <property name="text">
         <string>Generate reports for the containing folder</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rdGenerateReportsForSpecific">
        <property name="text">
         <string>Generate reports for specific folder/view</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QLineEdit" name="lineEditReportsForSpecific"/>
        </item>
        <item>
         <widget class="QToolButton" name="btnOpenFolderProperties">
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="../../qHostMonitor.qrc">
            <normaloff>:/img/foldersTree.png</normaloff>:/img/foldersTree.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>16</width>
            <height>16</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="btnSelectFolder">
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="../../qHostMonitor.qrc">
            <normaloff>:/img/browse.png</normaloff>:/img/browse.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>16</width>
            <height>16</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="lblRecursiveMode">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Recursive mode:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cmbRecursiveMode">
       <item>
        <property name="text">
         <string>Single folder</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Folder + all subfolders</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Folder + all subfolders/views with inherited settings</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="chkUseReportPool">
     <property name="text">
      <string>Use report pool</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../qHostMonitor.qrc"/>
  <include location="../../qHostMonitor.qrc"/>
 </resources>
 <connections/>
</ui>
